tests/TestCase.php

```diff
@@ -16,10 +16,31 @@
      */
     public function createApplication()
     {
+        putenv('DB_CONNECTION=sqlite_test');
+
         $app = require __DIR__.'/../bootstrap/app.php';
 
         $app->make(Illuminate\Contracts\Console\Kernel::class)->bootstrap();
 
+        $mockTwilioClient = Mockery::mock(\Twilio\Rest\Client::class)->makePartial();
+        $mockTwilioClient->messages = Mockery::mock();
+        $twilioNumber = config('services.twilio')['number'];
+        $mockTwilioClient->messages->shouldReceive('create');
+        $app->instance(\Twilio\Rest\Client::class, $mockTwilioClient);
+        
         return $app;
     }
+
+    public function setUp()
+    {
+        parent::setUp();
```

Comment:
You shouldn't migrate the database each time a test runs. If you need to leave data untouched you can mark your tests with `use DatabaseTransactions` https://laravel.com/docs/5.2/testing#resetting-the-database-after-each-test you can also migrate for every test like the link shows, but that shouldn't be necessary  if you use transactions.